### PR TITLE
DB permission should be update when starting  mangoDB on  Ubuntu16.04

### PR DESCRIPTION
**Pull request by LBTINGLB**
You need to ensure that your ‘mongo’ user should has ownership of that file
Job for mongod.service failed because the control process exited with error code.

### CODE REVIEW

**Comment by LBTINGLB** [comment]
@jeffbaumes @manthey Would you please kindly review this MR? 

Haocheng

File: docs/prerequisites.rst [hunk 1]
```diff
@@ -81,6 +81,7 @@ With Ubuntu 16.04, create a systemd init script at
 
 and start it with: ::
 
+  sudo chmod 600 var/lib/mongodb/
```
Comment (manthey) [inline docs/prerequisites.rst]: Shouldn't this be `/var/lib/mongodb` (with a leading slash)?

And, if we are updating information on installing Mongo, should we update the key server and apt repository information to the current values?

Comment (LBTINGLB) [inline docs/prerequisites.rst]: Nice catch Manthey. I was using this command from root path so it worked.

File: docs/prerequisites.rst [hunk 2]
```diff
@@ -81,6 +81,7 @@ With Ubuntu 16.04, create a systemd init script at
 
 and start it with: ::
 
+  sudo chmod 600 /var/lib/mongodb/
```
Comment (zachmullen) [inline docs/prerequisites.rst]: Is there a rationale for removing all permissions from other users of the system? If not, I'd suggest just

    sudo chmod u+rw /var/lib/mongodb/

Also, does this need to be applied recursively? If so add `-R` to the command.

**Comment by zachmullen** [comment]
@LBTINGLB we are planning to remove low-level specifics about mongo installation from our own docs due to that being out of scope, so I'm closing this for now.

File: docs/prerequisites.rst (hunk 2 above)
Comment (LBTINGLB) [inline docs/prerequisites.rst]: If I remember correctly, I think so.

**Comment by LBTINGLB** [comment]
@zachmullen Sorry for the late reply. Got it.

**Pull request closed without merging**